infrastructure/evault-provisioner/README.md

```
@@ -0,0 +1,188 @@
+
+2. **Evault Task**:
+   - Runs the evault application
+   - Connects to Neo4j via localhost
+   - Uses dynamic port allocation
+   - 512MB memory allocation
+   - Depends on Neo4j task
+
+## Environment Variables
+
+- `PORT` - Port to run the API on (default: 3000)
+- `NOMAD_ADDR` - Nomad API address (default: http://localhost:4646)
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)